libs/datasets/timeseries.py

```diff
@@ -310,6 +311,18 @@ def timeseries_wide_dates(self) -> pd.DataFrame:
         )
         return timeseries_wide
 
+    @lru_cache(maxsize=None)
+    def timeseries_wide_dates_variable_first(self) -> pd.DataFrame:
```

Comment:
fwiw, I think this might be more clear as an argument on `timeseries_wide_dates`, maybe:
```
def timeseries_wide_dates(self, variable_first_in_index=False)

```